### PR TITLE
ZER-53.1 Create tests for login page

### PR DESCRIPTION
**Pull request by VictoriaBryz**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: spec/features/visit_calculator_spec.rb [hunk 1]
```diff
@@ -3,9 +3,26 @@
...
+  let(:user) { create(:user) }
   before { create(:calculator) }
+
   it 'visits calculator page' do
     visit '/calculators/diapers-calculator'
     expect(page).to have_content 'receive email messages'
   end
+
+  it "visits calculator page and open log_in page" do
+    visit '/calculators/diapers-calculator'
```
[the remark this answers is not in the document]
Comment (VictoriaBryz) [reply spec/features/visit_calculator_spec.rb]: Thanks, I've changed it

File: spec/features/visit_calculator_spec.rb [hunk 2]
```diff
@@ -3,9 +3,26 @@
...
   before { create(:calculator) }
+
   it 'visits calculator page' do
     visit '/calculators/diapers-calculator'
     expect(page).to have_content 'receive email messages'
   end
+
+  it "visits calculator page and open log_in page" do
+    visit '/calculators/diapers-calculator' 
+    find('#log_in').click
```
[the remark this answers is not in the document]
Comment (VictoriaBryz) [reply spec/features/visit_calculator_spec.rb]: Thanks, I've changed it

File: spec/features/visit_calculator_spec.rb [hunk 3]
```diff
@@ -3,9 +3,26 @@
...
+    expect(page).to have_content 'Log in'
```
[the remark this answers is not in the document]
Comment (VictoriaBryz) [reply spec/features/visit_calculator_spec.rb]: Thanks, I've changed it

File: spec/features/visit_calculator_spec.rb [hunk 4]
```diff
@@ -3,9 +3,26 @@
...
+  end
+
+  it "when sign in with correct login and password" do
```
[the remark this answers is not in the document]
Comment (VictoriaBryz) [reply spec/features/visit_calculator_spec.rb]: Thanks, created a separate test

**Review by maksymst — APPROVED** [review]

**Review by PavloMS — APPROVED** [review]

**Pull request merged**